napari/layers/_shapes_layer/shape_list.py

```diff
@@ -657,3 +657,36 @@ def to_labels(self, labels_shape=None, shape_type=None):
                     labels[mask] = index[ind]
 
         return labels
+
+    def to_xml(self, shape_type=None):
+        """Returns a list of xml elements defining each shape according to the
+        svg specification. Z ordering of the shapes will be taken into account.
+        Passing a `shape_type` argument leads to only xml from that particular
+        `shape_type` being returned.
+
+        Parameters
+        ----------
+        shape_type : {'line', 'rectangle', 'ellipse', 'path', 'polygon'} |
```

Comment:
you don't need the `| None` if you have `, optional`

typically `, optional` is used to denote when an argument specifies a default value; said value may be `None` and if so will not require listing as an accepted type

Reply:
fixed